### ISSUE TITLE
CaliburnMicro projects missing files

### ISSUE THREAD

**Issue by ImGonaRot** [MvvmFx/MvvmFx#28]
The following CaliburnMicro projects are missing source files.

CaliburnMicro.Csla.WebGUI.Net4
CaliburnMicro.Csla.WebGUI.Net45
CaliburnMicro.Csla.WebGUI.Net46
CaliburnMicro.Csla.WinForms.Net45
CaliburnMicro.Csla.WinForms.Net46
CaliburnMicro.DialogExtensions.WebGUI.Net4
CaliburnMicro.DialogExtensions.WebGUI.Net45
CaliburnMicro.DialogExtensions.WebGUI.Net46
CaliburnMicro.DialogExtensions.WinForms.Net4
CaliburnMicro.DialogExtensions.WinForms.Net45
CaliburnMicro.DialogExtensions.WinForms.Net46
CaliburnMicro.WebGUI.Net4
CaliburnMicro.WebGUI.Net45
CaliburnMicro.WebGUI.Net46
CaliburnMicro.WinForms.Net45
CaliburnMicro.WinForms.Net46

**Comment by tfreitasleal** [comment]
Hi,

CaliburnMicro.DialogExtensions.WinForms.Net4 will have files once it's ready.
All the other projects use [linked files](https://msdn.microsoft.com/en-us/library/windows/apps/jj714082(v=vs.105).aspx). This is a nice feature that avoids the need to duplicate files.

On the different projects you define different constants (say WEBGUI or WINFORMS) according to the project target and on the source code you use compiler directives like
```
#if WEBGUI
    using FrameworkElement = Gizmox.WebGUI.Forms.Control;
#else
    using FrameworkElement = System.Windows.Forms.Control;
#endif
```

**Comment by ImGonaRot** [comment]
When I downloaded the entire code base, the sample CaliburnMicro.Samples.WinForms.VS2015.sln would not compile in VS 2015.
For example, "CaliburnMicro.WinForms.Net45.csproj" has compile includes such as (shared link)
    <Compile Include="..\CaliburnMicro.WinForms\Action.cs">
      <Link>Action.cs</Link>
    </Compile>
But there is no folder in the entire code base called "CaliburnMicro.WinForms".
I had to create this folder on my machine and then copy the files from "CaliburnMicro.WinForms.Net4" into the "CaliburnMicro.WinForms" folder.
I agree that they are "shared" linked files but some of the links are wrong / pointing to missing folders / files.
